tests/supertest/requestSender.spec.ts

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import express from 'express';
+import { expectTypeOf } from 'expect-type';
+import bodyParser from 'body-parser';
+import { RequestSender } from '../../src/requestSender/types';
+import { createRequestSender } from '../../src/requestSender/requestSender';
+import { paths, operations } from './types';
+
+describe('requestSender', () => {
+  let expressApp: express.Application;
+  let requestSender: RequestSender<paths, operations>;
+
+  beforeEach(async function () {
```

Comment:
why express init occures each test and not once per module?

Reply:
because each test we add routes with assertions and we dont want them to conflict